tuxemon/core/components/player.py

```diff
@@ -763,7 +763,7 @@ def move(self, tile_size, time_passed_seconds, game):
         collision_dict = {}
 
         # Get all the NPC's tile monsters_in_play so we can check for collisions.
-        for npc in game.npcs:
+        for slug,npc in game.npcs:

```

Comment:
Refer to my other comment [here](https://github.com/Tuxemon/Tuxemon/pull/203#discussion_r73624346) about this line.